tests/lib/rules/lines-around-comment.js

```diff
@@ -317,8 +317,195 @@ ruleTester.run("lines-around-comment", rule, {
                 allowBlockEnd: true
             }],
             ecmaFeatures: {classes: true}
-        }
+        },
+
+        // check for object start comments
```

Comment:
Just glanced quickly at the contributing documentation and didn't find it, what is the preferred style for multi-line strings? Thanks.